pom.xml

```diff
@@ -152,6 +153,7 @@
         <module>expression-language-tck</module>
         <module>bean-validation-tck</module>
         <module>persistence-tck</module>
+        <module>mail-tck</module>
```

Comment:
I'd rename this mail-platform-tck - at some point we'll probably want to port over the standalone mail TCK